capnslog/formatters.go

```diff
@@ -28,6 +29,28 @@ type Formatter interface {
 	Flush()
 }
 
+func NewSimpleFormatter(w io.Writer) *SimpleFormatter {
+	return &SimpleFormatter{
+		w: bufio.NewWriter(w),
```

Comment:
Talking about the field `w` on the `SimpleFormatter` struct - e.g. this won't work:
``
import "cpnslog"
f := capnslog.NewSimpleFormatter(bytes.Buffer{})
fmt.Println(f.w) // can't access w from outside package!
```